ePBFT/src/pbft/libbyz/test/test_ledger_replay.cpp

```diff
@@ -7,13 +7,14 @@
 #include "Node.h"
 #include "Replica.h"
 #include "Request.h"
+#include "consensus/ledgerenclave.h"
 #include "host/ledger.h"
 #include "network_mock.h"
 
 #include <cstdio>
 #include <doctest/doctest.h>
 
-static constexpr size_t TOTAL_REQUESTS = 1050;
+static constexpr size_t TOTAL_REQUESTS = 30;
```

Comment:
Why this change?

Reply:
Each entry has a size of 33094 and if we have 1050 requests that exceeds the ringbuffer circuit size. I could increase the circuit size, but this also makes the test quicker.

Reply:
Would you mind adding a comment for this?

Reply:
Actually, perhaps `circuit_size_shift` (or just `circuit_size`) could depend on `TOTAL_REQUESTS`? And we keep `TOTAL_REQUESTS` small for the reasons you mentioned?

Reply:
lemme try that out :)